src/Components/ContainerObjectInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Gui\Components;
+
+interface ContainerObjectInterface extends LazarusObjectInterface
+{
+    public function appendChild(VisualObjectInterface  $object);
```

Comment:
Put the summary here, and in the implementation use @inheritdoc for all methods.